### DISCUSSION TITLE
How to route different namespace/container logs to different indexes?

### DISCUSSION THREAD

**Discussion by suresh0204** [signalfx/splunk-otel-collector-chart#566] [Q&A] [answered]
Can someone please give an example of how to route different namespace/container logs to different indexes?

**Comment by atoulme** [comment, marked as answer]
See https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md#managing-log-ingestion-by-using-annotations for a documentation to achieve this.

**Reply by suresh0204** [reply]
Please let me know where we can configure this in values.yaml without running any command?

**Reply by jvoravong** [reply]
This example shows one way.
https://github.com/signalfx/splunk-otel-collector-chart/tree/main/examples/splunk-enterprise-index-routing
